packages/engine/src/avatar/functions/moveAvatar.ts

```diff
@@ -140,9 +142,48 @@ export const updateReferenceSpaceFromAvatarMovement = (movement: Vector3) => {
 
 const _additionalMovement = new Vector3()
 
+/**
+ * Avatar movement via click/pointer position
+ */
+
+const minimumDistanceSquared = 0.5 * 0.5
+const walkPoint = new Vector3()
+export const applyAutopilotInput = (entity: Entity) => {
+  const markerState = getState(AutopilotMarker)
+
+  const controller = getComponent(entity, AvatarControllerComponent)
+  if (!controller || markerState.walkTarget.value == undefined) return
```

Comment:
can do `!markerState.walkTarget.value` instead of `== undefined`

Reply:
Ya, no idea why I used !controller but then did == undefined immediately after.